tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/upgrade/Config.java

```diff
@@ -0,0 +1,34 @@
+package org.alfresco.elasticsearch.upgrade;
```

Comment:
Why creating second similar Interface to already created inner one? 
https://github.com/Alfresco/acs-packaging/pull/2234/files#:~:text=public%20interface-,ImagesConfig,-%7B

Reply:
It was added to decouple the upgrade tests components AcsEnv, Elasticsearch, ... from other parts of the system. In addition it makes it easier to run on a different version of docker images without changing some maven properties or env files.